public/app/features/serviceaccounts/components/ServiceAccountTokensTable.tsx

```diff
@@ -40,7 +39,12 @@ export const ServiceAccountTokensTable: FC<Props> = ({ tokens, timeZone, onDelet
                 <td>{formatDate(timeZone, key.created)}</td>
                 {contextSrv.hasPermission(AccessControlAction.ServiceAccountsDelete) && (
                   <td>
-                    <DeleteButton aria-label="Delete service account token" size="sm" onConfirm={() => onDelete(key)} />
+                    <DeleteButton
+                      aria-label="Delete service account token"
```

Comment:
```suggestion
                      aria-label={`Delete service account token ${key.name}`
```
Helps to be more descriptive here otherwise there will be list of button with some aria-labels.